Gemfile

```diff
@@ -21,7 +21,9 @@ gem 'jbuilder', '~> 2.7'
 # gem 'redis', '~> 4.0'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-
+gem 'devise', '~> 4.8'
+gem 'semantic-ui-sass'
+gem 'jquery-rails'
```

Comment:
Since we have webpacker, I would like using webpack for adding frontend features